lib/replicache.dart

```diff
@@ -135,32 +134,33 @@ class Replicache implements ReadTransaction {
     String name = '',
     String dataLayerAuth = '',
     String batchUrl = '',
+    @required RepmInvoke repmInvoke,
   }) async {
     final rep = _ReplicacheTest._new(
       remote,
       name: name,
       dataLayerAuth: dataLayerAuth,
       batchUrl: batchUrl,
+      repmInvoke: repmInvoke,
     );
     await rep._opened;
+    await rep._root;
     return rep;
   }
 
   /// Sets the level of verbosity Replicache should log at.
   static setLogLevel(LogLevel level) {
     globalLogLevel = level;
-    _staticInvoke(
-        '',
-        'setLogLevel',
-        {
+    _staticInvoke('', 'setLogLevel',
```

Comment:
I landed the change to make this a getter/setter so this needs to be rebased. Also, doesn't this require the invoke param?

Reply:
setters cannot have extra params `x = y` and all that.

We would need to make this a method too but I'm not sure I care enough. We can put it on the ReplicacheTest instance... Which by the way I'm going to move to lib/src/ in a follow up